router/router.go

```diff
@@ -46,6 +46,7 @@ type targetPassThroughAuth struct {
 	tokenExpires time.Time              // Expiry date of 'token'
 	tokenMap     map[string]interface{} // Map from username to token. For user-specific sessions with another machine.
 	tokenLock    map[string]bool        // If an entry exists in here for a username, then we are busy trying to log that user in.
+	filterMap    map[string]string      // YF module filter applied per user
```

Comment:
targetPassThroughAuth is shared by Yellowfin and Pure config. It would be better to include the yellowfin-specific stuff in a yellowfin-specific struct, which is stored as the "value" in tokenMap.